From 1901714988254b9098d299768bbfd6331c2d1ee4 Mon Sep 17 00:00:00 2001
From: Christian Lamparter <chunkeey@googlemail.com>
Date: Thu, 24 Nov 2011 00:41:28 +0100
Subject: [PATCH 37/63] carl9170 firmware: fix minute errors

 * typos in comments

 * inline static vs. static inline

 * line too long

Signed-off-by: Christian Lamparter <chunkeey@googlemail.com>
---
 carlfw/include/wol.h |   14 +++++++++-----
 carlfw/src/cmd.c     |    4 ++--
 2 files changed, 11 insertions(+), 7 deletions(-)

--- a/carlfw/include/wol.h
+++ b/carlfw/include/wol.h
@@ -38,24 +38,28 @@ struct ieee80211_hdr;
 
 void wol_prepare(void);
 void wol_janitor(void);
-void wol_rx(const unsigned int rx_filter __unused, const struct ieee80211_hdr *hdr __unused, const unsigned int len __unused);
+void wol_rx(const unsigned int rx_filter __unused,
+	    const struct ieee80211_hdr *hdr __unused,
+	    const unsigned int len __unused);
 void wol_cmd(const struct carl9170_wol_cmd *cmd);
 
 #else
 
-inline static void wol_cmd(const struct carl9170_wol_cmd *cmd __unused)
+static inline void wol_cmd(const struct carl9170_wol_cmd *cmd __unused)
 {
 }
 
-inline static void wol_prepare(void)
+static inline void wol_prepare(void)
 {
 }
 
-inline static void wol_janitor(void)
+static inline void wol_janitor(void)
 {
 }
 
-inline static void wol_rx(const unsigned int rx_filter __unused, const struct ieee80211_hdr *hdr __unused, const unsigned int len __unused)
+static inline void wol_rx(const unsigned int rx_filter __unused,
+			  const struct ieee80211_hdr *hdr __unused,
+			  const unsigned int len __unused)
 {
 }
 #endif /* CONFIG_CARL9170FW_WOL */
--- a/carlfw/src/cmd.c
+++ b/carlfw/src/cmd.c
@@ -125,7 +125,7 @@ void handle_cmd(struct carl9170_rsp *res
 		resp->hdr.len = 0;
 		disable_key(&cmd->disablekey);
 		break;
-#endif /* CONFIG_CARL9170FW_SECURIT_ENGINE */
+#endif /* CONFIG_CARL9170FW_SECURITY_ENGINE */
 
 #ifdef CONFIG_CARL9170FW_RADIO_FUNCTIONS
 	case CARL9170_CMD_FREQUENCY:
@@ -145,7 +145,7 @@ void handle_cmd(struct carl9170_rsp *res
 		fw.phy.psm.state = le32_to_cpu(cmd->psm.state);
 		rf_psm();
 		break;
-#endif /* CONFIG_CARL9170FW_RADIO_FUNCTIOS */
+#endif /* CONFIG_CARL9170FW_RADIO_FUNCTIONS */
 
 	default:
 		BUG("Unknown command %x\n", cmd->hdr.cmd);
